src/main/java/uk/gov/justice/digital/nomis/jpa/entity/CaseloadAgencyLocation.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.nomis.jpa.entity;
+
+import lombok.*;
```

Comment:
Not a fan of wildcards :/

Reply:
But all our devs use the standard IntelliJ Ultimate optimiser upon checkin - > 5 statics are grouped as a wildcard.  Don't want to have a "special" set of rules just for this project.  Consistency is more important.